src/js/components/RadioButtonGroup/__tests__/__snapshots__/RadioButtonGroup-test.js.snap

```diff
@@ -408,7 +408,12 @@ exports[`RadioButtonGroup boolean options 1`] = `
           </svg>
         </div>
       </div>
-      <span>
+      <span
+        className=""
+        onBlur={[Function]}
```

Comment:
This is probably from passing ...rest to the StyledRadioButtonLabel. Looking to see how other components handle this...